network/topology/randomizedTopology_test.go

```diff
@@ -52,26 +50,24 @@ func (suite *RandomizedTopologyTestSuite) SetupTest() {
 	// mocks state for collector nodes topology
 	suite.state, suite.clusters = MockStateForCollectionNodes(suite.T(),
 		suite.all.Filter(filter.HasRole(flow.RoleCollection)), uint(nClusters))
-
-	suite.subMngr = MockSubscriptionManager(suite.T(), suite.all)
```

Comment:
Channels list is provided directly to `GenerateFanout` of the topology instance. No further need to add it as a member variable through its factory.